lib/asset_sync/config.rb

```diff
@@ -242,7 +250,8 @@ def load_yml!
       self.google_storage_access_key_id     = yml["google_storage_access_key_id"] if yml.has_key?("google_storage_access_key_id")
       self.google_json_key_string           = yml["google_json_key_string"] if yml.has_key?("google_json_key_string")
       self.existing_remote_files  = yml["existing_remote_files"] if yml.has_key?("existing_remote_files")
-      self.gzip_compression       = yml["gzip_compression"] if yml.has_key?("gzip_compression")
+      self.compression            = yml["compression"] if yml.has_key?("compression")
+      self.compression            = 'gz' if yml.has_key?("gzip_compression") and yml.has_key?("gzip_compression")
```

Comment:
- New option > old option = should be set later
- Why repeated `yml.has_key?("gzip_compression")` and the usage of `and`???
```suggestion
      self.compression            = 'gz' if yml.has_key?("gzip_compression")
      self.compression            = yml["compression"] if yml.has_key?("compression")
```